### ISSUE TITLE
Loss=nan with all learning rates

### ISSUE THREAD

**Issue by Taikakim** [TheLastBen/fast-stable-diffusion#2464]
I'm running the Runpod LoRA trainer, new instance created today. I am trying to train on 26 1024x1024 images with captions, LoRA dimension 256. I've now tried a bunch of settings, but what matter I do, I'm getting loss=NaN from the very start, and LoRAs only create black images. I'm running on a 24GB RTX 3090 instance. This is weird since I just the other day did several good models with a slightly older version of the notebook.

**Comment by TheLastBen** [comment]
the default is 1e-6 not 1e-4

**Comment by Taikakim** [comment]
That is very strange why it was on that value. Now it is working again, I've been trying to set the TE epochs so that there's around 300 total steps. I am a bit puzzled how the TE value was like that, maybe it was some unconscious editing by myself which I was not aware of :)

The Runpod is very comfortable by the way in how fast the notebook starts when everything does not have to be installed from scracth every time it is run.
